js/data.js

```diff
@@ -6,3 +6,10 @@ var data = {
   editing: null,
   nextEntryId: 1
 };
+
+window.addEventListener('beforeunload', function (event) {
+  var dataJson = JSON.stringify(data);
+  var entriesJson = JSON.stringify(data.entries);
+  localStorage.setItem('data', dataJson);
+  localStorage.setItem('entries', entriesJson);
```

Comment:
Your entries should be stored on the `entries` property of the `data` object. Everything that is stored should be within the `data` object so you only have to save one thing to local storage.

Reply:
https://user-images.githubusercontent.com/88458577/132558725-d9c47710-0610-453c-9630-23e927358101.mov

Reply:
Fixed